te/GET_AND_RUN_DOCKER_IMAGE.py

```diff
@@ -118,9 +120,11 @@ def __init__(self, pathToDocker, ip, nginxPort, stat_collect_interval, stat_dump
                 'repo_name'           : 'tedp',
                 'run_cmd'             : "docker run --privileged --cap-add=SYS_PTRACE --security-opt " \
                                         "seccomp=unconfined -v /tmp/:/te_host/ -v $HOME:/te_root/ " \
-                                        "-v /var/run/netns:/var/run/netns " \
+                                        "-v $HOME/.ssh/:/root/.ssh/ -v /var/run/netns:/var/run/netns " \
```

Comment:
Refer to the PR title where in we try to fallback to password less system default cert based auth if no password is given. To facilitate this we need to mount the ~/.ssh/ folder within the container